src/Common/Handlers/PublishPackageHandler.cs

```diff
@@ -118,7 +127,8 @@ private async Task<bool> PackageExists(string fileName)
                 {
                     Session.LogDebug("Package does not exist");
                 }
-            } else
+            }
+            else
             {
                 Session.LogDebug($"Could not get data from file name ({fileName}) to validate if package exists.");
```

Comment:
![image](https://github.com/user-attachments/assets/02b69089-1030-4e2c-b8f5-0801c4bfb227)
We should throw somewhere if the version is not respecting our definition allowed, otherwise this situation occurs, we can upload with and invalid version

Reply:
![image](https://github.com/user-attachments/assets/e22395dd-e342-4d88-ad36-faad26337687)
Working now